docs/UserGuide.md

```diff
@@ -113,7 +114,7 @@ Format: `add-wedding w/NAME & NAME v/VENUE d/DATETIME`
 
 <box type="tip" seamless>
 
-**Tip:** Datetime must be a valid date in the format of dd/MM/yyyy
+**Tip:** Datetime must be a valid date in the format of **dd/MM/yyyy**
```

Comment:
We should change all datetime to time due to the refactoring

Reply:
will do